OpenSkyPlusConfiguration/IOpenSkyPlusConfiguration.cs

```diff
@@ -1,20 +1,22 @@
-namespace OpenSkyPlus;
-
-public interface IOpenSkyPlusConfiguration
-{
-    public struct AppSettings(
-        string logPath,
-        string logLevel,
-        string launchMonitor,
-        string launchMonitorAppPath,
-        string pluginPath,
-        string shotConfidence)
-    {
-        public string LogPath { get; set; } = logPath;
-        public string LogLevel { get; set; } = logLevel;
-        public string LaunchMonitor { get; set; } = launchMonitor;
-        public string LaunchMonitorAppPath { get; set; } = launchMonitorAppPath;
-        public string PluginPath { get; set; } = pluginPath;
-        public string ShotConfidence { get; set; } = shotConfidence;
-    }
+namespace OpenSkyPlus;
+
+public interface IOpenSkyPlusConfiguration
+{
+    public struct AppSettings(
+        string logPath,
+        string logLevel,
+        string launchMonitor,
+        string launchMonitorAppPath,
+        string pluginPath,
+        string shotConfidence,
+        bool refreshConnectionAfterModeSwitch)
+    {
+        public string LogPath { get; set; } = logPath;
+        public string LogLevel { get; set; } = logLevel;
+        public string LaunchMonitor { get; set; } = launchMonitor;
+        public string LaunchMonitorAppPath { get; set; } = launchMonitorAppPath;
+        public string PluginPath { get; set; } = pluginPath;
+        public string ShotConfidence { get; set; } = shotConfidence;
+        public bool RefreshConnectionAfterModeSwitch { get; set; } = refreshConnectionAfterModeSwitch;
```

Comment:
Why make this configurable? It would either work or not work, no?

Reply:
Funny you should ask because it was absolutely working for me without the refresh, then suddenly it stopped and I have no idea what changed. We can try it again without the refresh and see what happens?

Reply:
Yep, good idea.  If anything there will be a setting to re-enable it in case there are suspicions that it might be related to that.